server/lib/init_log.rb

```diff
@@ -0,0 +1,24 @@
+require "fusor/multilog"
+
+module Fusor
+  @default_log_file = "/var/log/foreman/fusor.log"
+
+  def self.log
+    @log ||= MultiLogger.new(Rails.logger)
+  end
+
+  def self.log_change_deployment(deployment = nil)
+    self.log
+
+    #@log.detach_all
+
+    if Rails.env.production?
+      if deployment.nil?
+        @log.attach(@default_log_file)
+      else
+        Dir.mkdir("/var/log/foreman/#{deployment.name}-#{deployment.id}") unless File.exist?("/var/log/foreman/#{deployment.name}-#{deployment.id}")
+        @log.attach("/var/log/foreman/#{deployment.name}-#{deployment.id}/deployment.log")
```

Comment:
I wouldn't hardcode this to `/var/log/foreman`. If you use `File.join(Rails.root, 'log')`, this will work in all environments.

Also, I raised [an issue about using the deployment.name in the filename in a previous PR](https://github.com/fusor/fusor/pull/485#discussion_r45390172). Not sure if you saw that.